RecastDemo/Source/Sample.cpp

```diff
@@ -35,6 +35,32 @@
 #	define snprintf _snprintf
 #endif
 
+unsigned int sampleAreaToCol(unsigned int area)
+{
+	switch(area)
+	{
+	// Ground (0) : brown
+	case SAMPLE_POLYAREA_GROUND: return duRGBA(125, 125, 0, 255);
+	// Water : blue
+	case SAMPLE_POLYAREA_WATER: return duRGBA(0, 0, 255, 255);
+	// Road : dark grey
+	case SAMPLE_POLYAREA_ROAD: return duRGBA(80, 80, 80, 255);
+	// Door : cyan
+	case SAMPLE_POLYAREA_DOOR: return duRGBA(0, 255, 255, 255);
+	// Grass : green
+	case SAMPLE_POLYAREA_GRASS: return duRGBA(0, 255, 0, 255);
+	// Jump : yellow
+	case SAMPLE_POLYAREA_JUMP: return duRGBA(255, 255, 0, 255);
+	// Unexpected : red
+	default: return duRGBA(255, 0, 0, 255);
+	}
+}
+
+unsigned int SampleDebugDraw::polyToCol(const struct dtPoly* poly)
```

Comment:
Of course !
My bad, too much used to rely on the override keyword.
Should be ok now.